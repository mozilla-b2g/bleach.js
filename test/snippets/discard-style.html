<style type="text/css">p { color: red; background-color: blue; }
blockquote { color: pink; }</style>I am the <span>a<span>ctua</span>l</span> content.