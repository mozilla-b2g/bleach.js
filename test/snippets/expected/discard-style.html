I am the actual content.